app/models/item.rb

```diff
@@ -3,10 +3,13 @@ class Item < ApplicationRecord
   has_many :invoice_items
   has_many :invoices, through: :invoice_items
 
+  enum status: [:Disabled, :Enabled]
```

Comment:
does it matter if these values are capitalized?

Reply:
this post suggest camel case naming convention for enums. so lowercase. https://stackoverflow.com/questions/40152468/enums-in-rails-uppercase-or-lowercase